.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+sudo: false
+
```

Comment:
`SYMFONY_PHPUNIT_DIR` is here for that reason. Not sure if we can have a better default.